numpy/linalg/tests/test_linalg.py

```diff
@@ -845,6 +846,111 @@ def test_empty(self):
         assert_equal(norm(array([], dtype=self.dt)), 0.0)
         assert_equal(norm(atleast_2d(array([], dtype=self.dt))), 0.0)
 
+    def test_vector_types(self):
```

Comment:
Having `return_types` in there helps. I usually have separate tests for exceptions, values, types, etc. Since we are just checking the return type here being explicit in the name is good.